virtwho/runner.py

```diff
@@ -46,19 +47,19 @@ def run_virtwho_cli(self,
             test interval function.
         :return: tty_output
         """
-        cmd = 'virt-who '
+        cmd = "virt-who "
```

Comment:
'xxx' is more better format, any issue for it?

Reply:
No issue, just want to make a uniform. And yes, it looks better to use '' than "" somewhere.